prog/dftb+/lib_elecsolvers/elsisolver.F90

```diff
@@ -490,9 +490,8 @@ subroutine supportedVersionNumber(major, minor, patch, dateStamp)
       isSupported = .false.
     elseif (major == 2 .and. minor < 6) then
       isSupported = .false.
-    elseif (major == 2 .and. minor == 6 .and. patch < 0) then
-      ! no need to check patch in this case, as library must be >= 2.6.0, but just in case of a
-      ! negative value
+    elseif (major == 2 .and. minor == 6 .and. all(patch == [0,1])) then
+      ! library must be 2.6.{0,1}
```

Comment:
`all(patch == [0, 1])` will be always `.false.`. Why don't we leave this away and keep only the checks ensuring not being older than 2.6.0?

Reply:
Replaced with any() instead of all()